elifetools/parseJATS.py

```diff
@@ -12,7 +12,7 @@
 from utils import unicode_value
 
 def parse_xml(xml):
-    return BeautifulSoup(xml, ["lxml", "xml"])
+    return BeautifulSoup(xml, "lxml-xml")
```

Comment:
I'm only a casual user of `beautifulsoup`, I know this is the name of the parser, but couldn't find documentation that lists the possible values

Reply:
Tests seem to like it though

Reply:
There are some options listed at https://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-a-parser

I believe when we upgraded BeautifulSoup in the requirements.txt, this is the recommended value for that version, as opposed to the former value.